benchmarks/docker/django_simple/config/settings/production.py

```diff
@@ -0,0 +1,80 @@
+from .base import *  # noqa
+from .base import env
+
+# GENERAL
+# ------------------------------------------------------------------------------
+# https://docs.djangoproject.com/en/dev/ref/settings/#secret-key
+SECRET_KEY = "6xHyHMIwEJy8BR5hatlmaI7OSBpHMi9yyNuTxB4rd9lCdNOiAbEdpsucEF1EU9tO"
```

Comment:
can we use an obviously bogus name? like `SECRET_KEY = "SECRET_KEY"` or something that won't look like a real secret